README.md

```diff
@@ -120,3 +129,27 @@ npm run changeset
 ```
 
 Make sure to add the change log found in the `.changeset` directory to the PR.
+
+## Run dev server
+
+To run a development server:
+
+```
+npm run dev -w @lookit/<name of package>
+```
+
+When the server has started, you should see something very similar to `<script src="http://127.0.0.1:10001/index.browser.js"></script>` printed out. Add this html to `web/templates/web/jspsych-study-detail.html` in the Django lookit api project to test the package in your local development environment.
+
+### Serve multiple packages
+
+The above command will serve a single package and wait for changes. If you need to serve multiple packages locally, you can open separate terminals for each package and run the `npm run dev` command in each. Another option is to install [Honcho](https://github.com/nickstenning/honcho) and write a Procfile to serve/watch multiple packages in the same terminal. 
+
+Create a file called `Procfile` in the root project directory, and list the `npm run dev` commands for each package that you want to serve, preceded by the label you want to give it (to identify the print statements associated with each package in the terminal). 
+
+```
+lookit-initjspsych: npm run dev -w @lookit/lookit-initjspsych
+lookit-api: npm run dev -w @lookit/lookit-api
+lookit-helpers: npm run dev -w @lookit/lookit-helpers
+```
+
+This method is optional (Honcho should not be added to the project dependencies, and Procfile has been added to our gitignore).
```

Comment:
@okaycj is this section ok with you?

Reply:
This looks good. But if we both want the honcho `Procfile`should we just commit it to the repository?  But maybe we've talked about this too long and just need to move on.

Reply:
Hmm, true. Let's leave it out for now. I think that after the initial set of packages is published, we'll probably just be working on a single local package at a time and the rest will come from a CDN, so it'll be fine to just use `npm run dev`. We can always commit this later if we're using it a lot.